src/layouts/EditPage.jsx

```diff
@@ -130,6 +130,10 @@ const EditPage = ({ match }) => {
         siteName,
         DOMCSPSanitisedHtml
       )
+      DOMPurify.removed = DOMPurify.removed.filter(
+        (el) =>
+          !(el.element.tagName === "SCRIPT" && el.element.hasAttribute("src"))
+      )
```

Comment:
correct me if i'm wrong but one of the means of xss violation is through unvalidated `script` tags, which is what we seem to be allowing here.

should we make this more specific - ie, to only allow the sites that you've listed in #893?

Reply:
The part where only the sites listed in #893 are allowed as the source URL is handled by the `checkCSP()` function. My assumption is that if it is prevented by the `checkCSP()` function, it will not reach this segment of the code.

If there is any inline script, that would also be blocked by CSP.

Reply:
Inline scripts are blocked from loading on the actual site due to the CSP, but I updated the code to block it from saving as well.